### ISSUE TITLE
Last Request Returning First Request

### ISSUE THREAD

**Issue by lcharbon** [ldabiralai/simulado#104]
Calling `lastRequest` returns the oldest request. Was expecting it to return the newest.

Is this possible? Will provide a minimal example to help with troubleshooting if necessary.

**Comment by ldabiralai** [comment]
Thanks @lcharbon, you raise a good point. I think this is a bug in simulado at the moment, but fixing this is a "breaking" change.

`lastRequest` uses `lastRequests` under the hood, I think ideally `lastRequests` should also return from **newest to oldest**, but at the moment it does **oldest to newest**, so it'd be good to change that too. I'll take some time to think about the correct solution and get this fixed for you in v4.

In the meantime, you could do the following:
```
const lastRequests = await Simulado.lastRequests('GET', '/path');
const lastRequest = lastRequests[lastRequests.length - 1]
```

**Comment by bdo** [comment]
Fixed by @ldabiralai in #105
